apollo-configservice/src/main/java/com/ctrip/framework/apollo/configservice/controller/NotificationControllerV2.java

```diff
@@ -111,10 +107,14 @@ public NotificationControllerV2(
     }
 
     DeferredResultWrapper deferredResultWrapper = new DeferredResultWrapper(bizConfig.longPollingTimeoutInMilli());
-    Set<String> namespaces = Sets.newHashSet();
-    Map<String, Long> clientSideNotifications = Maps.newHashMap();
+   
     Map<String, ApolloConfigNotification> filteredNotifications = filterNotifications(appId, notifications);
 
+    //appoint initialCapacity
```

Comment:
how about using the existing methods?

```java
    Set<String> namespaces = Sets.newHashSetWithExpectedSize(filteredNotifications.size());
    Map<String, Long> clientSideNotifications = Maps.newHashMapWithExpectedSize(filteredNotifications.size());
```

Reply:
it is great。

I didn't think of this idea， because  I don't use `guava` very often

Reply:
so, would you commit this code by yourself ?

At the same time, i find some other optimization 。because this `NotificationControllerV2.java` is so important that it cannot be too concerned

the PR is : #3145